src/subscription/data/details/selectors.js

```diff
@@ -0,0 +1,76 @@
+import { getConfig, getQueryParameters } from '@edx/frontend-platform';
+import { createSelector } from 'reselect';
+import Cookies from 'universal-cookie';
+
+import { storeName } from '../constants';
+
+export const localizedCurrencySelector = () => {
```

Comment:
Could this move to another file where the regular payment flow could also used since it's the same repeated code?

Reply:
moved under `src/payment/data/utils`